11ty/definitions/xenophobia.md

```diff
@@ -0,0 +1,22 @@
+---
+title: xenophobia
+slug: xenophobia
+defined: true
+excerpt:  hatred of and discrimination against people from other countries, their culture, and/or their religion that is rooted in the fear of people who are "different"
+speech: noun
+flag:
+  level: warning
+  text: content warning
+---
+
+hatred of and discrimination against people from other countries, their culture, and/or their religion that is rooted in the fear of people who are "different"
+
+## Issues
+
+Xenophobia when broken down translates to the "fear of strangers." It stems from a psychological point of view that there is an "ingroup" and an "outgroup" that exist within a community, which is known as [ingroup-outgroup bias](http://psychology.iresearchnet.com/social-psychology/group/ingroup-outgroup-bias/). Whether it's conscious or unconscious, ingroup-outgroup bias can lead to harmful results, including xenophobia.
```

Comment:
```suggestion
Xenophobia translates to the "fear of strangers" from Greek [_xenos_](https://www.etymonline.com/word/xeno-?ref=etymonline_crossreference) for guest, stranger, foreigner, refugee, guest-friend, one entitled to hospitality and [_-phobia_](https://www.etymonline.com/word/-phobia?ref=etymonline_crossreference) for panic or fear of.  It stems from a psychological point of view that there is an "ingroup" and an "outgroup" that exist within a community, which is known as [ingroup-outgroup bias](http://psychology.iresearchnet.com/social-psychology/group/ingroup-outgroup-bias/). Whether it's conscious or unconscious, ingroup-outgroup bias can lead to harmful results, including xenophobia.
```
As it reads, the last sentence here feels a little cyclical/self-referential? While I think that xenophobia is equally an outcome, it's a transitive one that leads to systemic bias and then systemic oppression. I think we need to paint this lineage a bit more clearly somehow.